clouddriver-ecs/src/integration/resources/testoperations/createServerGroupOperation-inputs-fargate-NewTargetGroup.json

```diff
@@ -0,0 +1,39 @@
+{
+  "account": "ecs-account",
+  "application": "ecs",
+  "availabilityZones": {
+    "us-west-2": [
+      "us-west-2a",
+      "us-west-2c"
+    ]
+  },
+  "capacity": {
+    "desired": 1,
+    "max": 1,
+    "min": 1
+  },
+  "cloudProvider": "ecs",
+  "computeUnits": 256,
+  "credentials": "ecs-account",
+  "dockerImageAddress": "nginx",
+  "ecsClusterName": "spinnaker-deployment-cluster",
+  "launchType": "FARGATE",
+  "networkMode": "aws-vpc",
+  "placementStrategySequence": [],
+  "reservedMemory": 512,
+  "stack": "integTestStack",
+  "freeFormDetails" : "detailTest",
+  "targetGroupMappings": [
+    {
+      "containerName" : "main",
+      "containerPort" : 80,
+      "targetGroup" : "spin-fargate-instanceTG"
+    }
+  ],
+  "source" : {
```

Comment:
what happens if we don't include this `source` arg, does it fail?

Reply:
Source is not required. It was added as a next part of the testing but not really required for this test.